SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -508,3 +535,43 @@ void USpatialClassInfoManager::QuitGame()
 	FGenericPlatformMisc::RequestExit(false);
 #endif
 }
+
+Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(const AActor* Actor) const
+{
+	check(Actor);
+	const AActor* ActorForRelevancy = Actor;
+	// bAlwaysRelevant takes precedence over bNetUseOwnerRelevancy - see AActor::IsNetRelevantFor
+	while (!ActorForRelevancy->bAlwaysRelevant && ActorForRelevancy->bNetUseOwnerRelevancy && ActorForRelevancy->GetOwner())
+	{
+		ActorForRelevancy = ActorForRelevancy->GetOwner();
+	}
+
+	if (ActorForRelevancy->bAlwaysRelevant)
+	{
+		return SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID;
+	}
+
+	if (GetDefault<USpatialGDKSettings>()->bEnableNetCullDistanceInterest)
+	{
+		Worker_ComponentId NCDComponentId = GetComponentIdForNetCullDistance(ActorForRelevancy->NetCullDistanceSquared);
+		if (NCDComponentId != SpatialConstants::INVALID_COMPONENT_ID)
+		{
+			return NCDComponentId;
+		}
+
+		const AActor* DefaultActor = Actor->GetClass()->GetDefaultObject<AActor>();
```

Comment:
Should we be looking at the default actor of `ActorForRelevancy`?

Reply:
Good shout